src/DataLayouts/broadcast.jl

```diff
@@ -377,9 +377,21 @@ end
 
 # inline inner column(::DataColumn) copy
 @inline function Base.copyto!(
-    dest::VF{S},
-    bc::Union{VF{S, A}, Base.Broadcast.Broadcasted{VFStyle{A}}},
-) where {S, A}
+    dest::T,
+    bc::VF{S, A},
+)::T where {S, A, T <: VF{S}}
```

Comment:
I am skeptical about this one, but maybe the force inline is hiding the implicit specialization.

Is the same result achieved keeping the Union but writing it this way where the return type is declared to be the `dest type`, ex: `copyto!()::T`?

Reply:
One TODO is to make `VFStyle{S, A}` as well (to mirror the datalayout)

Reply:
Reminder: we looked at this in a call and it actually isn't an issue?